integration/spark-common-cluster-test/src/test/scala/org/apache/carbondata/cluster/sdv/generated/PartitionTestCase.scala

```diff
@@ -1,520 +0,0 @@
-
```

Comment:
SDVSuites.scala is still pointing to this file. 
Need to remove from there also